strapi/strapi
# fix(content-releases): remove the error in the console when we delete a release

### What does it do?

We encountered an issue with the invalidateTags function when deleting a release. This arose because invalidateTags executed before redirecting to the list view. Consequently, a GET request to the recently deleted release was initiated, resulting in a 404 error. To address this, I replaced invalidateTags with an onQueryStarted async function. This change ensures that invalidateTags is dispatched only a few milliseconds after the successful deletion call, allowing time for the page to redirect to the list view.

### Why is it needed?

When deleting a release, we invalidate a corresponding tag, which serves as the cached value for the GET release call. This causes an immediate 404 error upon the subsequent GET request, as the release has already been deleted. 

### How to test it?

- create a new release
- open the details view of the release and delete the release (before confirming the action please open the devTool console)
- you will be redirected to the releases list view without error in the console and if you click the browser's back button a notification will appear with the message "Your request could not be processed. Please try again or open another release." 

### Related issue(s)/PR(s)

CS-583
